### ISSUE TITLE
Background in top of Templates folder

### ISSUE THREAD

**Issue by madsrh** [ubuntu/yaru#1989]
I know that per app tweaks isn't ideal, but IMHO the label in the Templates folder lacks discoverability which is key for it's usefulness. Opposed to the Trash folder, the top banner here acts as information and would benefit from a background.

WDYT?

![image](https://user-images.githubusercontent.com/3986894/75255230-e20d4500-57e1-11ea-96b2-f75ed6ef76bc.png)

**Comment by madsrh** [comment]
Okay I was pretty alone on this one 🙈

**Comment by friendlyapple** [comment]
@madsrh I think it's a good idea :)

**Comment by madsrh** [comment]
👍 @friendlyapple 😁

I tried this to do this with `infobar.info` but couldn't make it work. Is this even possible @Feichtmeier ?

**Comment by clobrano** [comment]
It is
![image](https://user-images.githubusercontent.com/2883614/75606199-ab3d7480-5aea-11ea-85fe-386537257662.png)

**Comment by clobrano** [comment]
Ah, "unfortunately" also the Thrash folder has an infobar
![image](https://user-images.githubusercontent.com/2883614/75606317-b644d480-5aeb-11ea-83f9-298b41bd41af.png)

**Comment by Feichtmeier** [comment]
Haha infobars again xD 
I think that yellow looks like that 3M post-its so I think it looks good for a info :+1: 
But what I don't like is that upstream just ignores the differences between the info bars: warning, info, error and so on

**Comment by madsrh** [comment]
> "unfortunately" also the Thrash folder has an infobar

Why is unfortunately inside a " " 😂

> Shall we go with your suggestion instead (#f3ef82)?

Sure or even lighter perhaps?

> upstream just ignores the differences between the info bars: warning, info, error and so on

So, you're both saying that it isn't possible?

**Comment by Feichtmeier** [comment]
No we had it once... back in time... I think... let me see...
https://github.com/ubuntu/yaru/issues/856#issuecomment-423826338

**Comment by clobrano** [comment]
It totally makes sense. We went through a plethora of ideas :D
Right now, upstream does not make any difference, however yellow might suit better to a warning message, even more in dark variant, where the yellow should be darker.

What do you think of this?
![nautilus-infobar](https://user-images.githubusercontent.com/2883614/75681668-5ccbd980-5c94-11ea-80eb-33e07a155d63.gif)

**Comment by Feichtmeier** [comment]
That looks def. better for an information
I think upstream has the most lazy implementation here with inverting the selected_bg_color :man_facepalming: 
If you change it would you change all three types like we had it once?

**Comment by clobrano** [comment]
I'm just changing the infobar in nautilus right now, mostly because we already saw that those bars strongly depends on the context. In Nautilus this grey can be fine, while in another app might not

**Comment by madsrh** [comment]
Just to be clear: are you saying that we can't safely style this only for Nautilus? If so, then let's not do this

**Comment by clobrano** [comment]
No, the other way around :smiley:. I suggest doing it only for Nautilus and it is safe to do it

**Comment by Feichtmeier** [comment]
For the templates folder this looks cool
For the trash I think it is a little bit more "dangerous" and the yellow fits, imo.

**Comment by clobrano** [comment]
They are the same kind of infobar

**Comment by Feichtmeier** [comment]
I think the backdrop could need a little improvement in the dark theme @clobrano 
Maybe just $backdrop_base_color like the rest of the window
![Screenshot from 2020-03-06 10-27-17](https://user-images.githubusercontent.com/15329494/76076304-7d4aab00-5f96-11ea-80da-76da297223f8.png)

**Comment by clobrano** [comment]
Like this?

![infobar](https://user-images.githubusercontent.com/2883614/76080017-4b414500-5fa6-11ea-9d8c-f9735e9ed13f.gif)

**Comment by madsrh** [comment]
Perfect 💪

**Comment by Feichtmeier** [comment]
:+1:
